csi/shell/check_bind_mounts.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I've found the most common way to check mount is the following way:

https://serverfault.com/questions/50585/whats-the-best-way-to-check-if-a-volume-is-mounted-in-a-bash-script